bin/milight

```diff
@@ -1,32 +1,199 @@
-#!/usr/bin/env ruby
+#!/usr/lib/env ruby
 # frozen_string_literal: true
 
 require "milight/v6"
+require "optparse"
+require "ipaddress"
```

Comment:
ipaddress is a separate gem. maybe use IPAddr which is standard Ruby (or skip validation at all)

Reply:
Ah cool ok - yes this would be better